publish_jenkins_console.py

```diff
@@ -7,11 +7,13 @@
 from collections import OrderedDict
 
 import requests
+import urllib
 from requests.exceptions import ConnectionError
 from jenkinsapi.jenkins import Jenkins
 
-# fpast url
-FPASTE_URL = 'https://paste.fedoraproject.org/api/paste/submit'
+APIKEY = urllib.urlencode({'apikey': '5uZ30dTZE1a5V0WYhNwcMddBRDpk6UzuzMu-APKM38iMHacxdA0n4vCqA34avNyt'})
+SERVER_URL = 'https://paste.centos.org'
```

Comment:
@FilipB Where do you use this variable? `SERVER_URL`

Reply:
That line is removed now